### PR TITLE
[10.3] Backport changes to big_file_upload_configuration.adoc from master

### PR DESCRIPTION
**Pull request by settermjd**
While investigating #2152, I found a number of changes to `modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc` in master that haven't been backported to 10.3. This PR backports all of the outstanding changes.

### CODE REVIEW

File: modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc [hunk 1]
```diff
@@ -97,61 +89,68 @@ When you have done run `asdismod` or updated `/etc/httpd/conf/httpd.conf`, resta
...
 
-Set the following two parameters inside `php.ini`, using your own
-desired file size values:
+Set the following two parameters inside `php.ini`, using your own desired file size values, as in the following example:
 
+[source]
 ----
 upload_max_filesize = 16G
 post_max_size = 16G
 ----
```
Comment (pako81) [inline modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc]: Not needed anymore

**Comment by pako81** [comment]
@settermjd 

Infos that apply to chunked uploads: PHP timeouts, Apache modules, temp directory size.

Infos that apply to non-chunked uploads (public links): PHP timeouts, Apache modules.

The "default maximum file size for uploads" info is obviously not valid anymore.

It makes probably sense to split the page up into two separated sections.

File: modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc [hunk 2]
```diff
@@ -46,33 +39,32 @@ PrivateTmp=false
...
-Set the following two parameters inside the corresponding php.ini file
-(see the *Loaded Configuration File* section of xref:configuration/general_topics/general_troubleshooting.adoc#php-version-and-information[PHP Version and Information]
-to find your relevant php.ini files) :
+Set the following two parameters inside the corresponding php.ini file (see the *Loaded Configuration File* section of xref:configuration/general_topics/general_troubleshooting.adoc#php-version-and-information[PHP Version and Information] to find your relevant php.ini files):
 
+[source,console]
 ----
 php_value upload_max_filesize = 16G
 php_value post_max_size = 16G
 ----
```
Comment (pako81) [inline modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc]: Not needed anymore.

Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc]: That should be kept, but corrected to be `[source,php]` for proper syntax highlighting.

File: modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc (hunk 1 above)
Comment (settermjd) [reply modules/admin_manual/pages/configuration/files/big_file_upload_configuration.adoc]: Will remove it. Thanks.

**Comment by settermjd** [comment]
Thanks for the review, @pako81. Changes made.

**Comment by settermjd** [comment]
Just removed more empty source blocks too, @pako81.

**Pull request merged**